.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### Set `develop` as target branch of this PR (delete this line before issuing the PR)
+
+## Description
```

Comment:
Please state that the test inclusion requirements as follows:

* any fix or enhancement SHOULD include relevant new tests or test updates, if any tests need updating. 
* a new platform support MUST include the relevant input files similar to what we have in tests/integration/sqcmds/*-input directories, along with the relevant tests in the tests/integration/sqcmds/*-samples dir. That list MUST include the all.yml file fully filled out.
* any new service (or table) addition MUST include comments about what network OS are supported (along with version) with this command along with test samples for those platforms and input files in the *-input dir

Reply:
Please provide similar comments about what's necessary in documentation.

Reply:
Done